### ISSUE TITLE
**预加载之前训练的模型**

### ISSUE THREAD

**Issue by jideee** [shaoshengsong/MobileNetV3-SSD#28]
你好 !
我按着步骤来 进行 预加载训练.
但是 训练时没有从Epoch150开始  而是直接又从0开始训练了   
请问这是怎么回事?  我应该怎么调.

cjd@jordan:~/car_FPN-G-F-MobileNetV3-SSD$ python3.6 train_ssd.py --datasets /home/cjd/ssd.pytorch/data/VOCdevkit/VOC2007 --validation_dataset /home/cjd/ssd.pytorch/data/VOCdevkit/VOC2007 --net mb3-ssd-lite --pretrained_ssd models_new/mb3-ssd-lite-Epoch-149-Loss-2.2059922739140037.pth --scheduler cosine --lr 0.01 --t_max 100 --validation_epochs 5 --num_epochs 300 --base_net_lr 0.0012 --batch_size 5
2020-07-15 17:08:40,220 - root - INFO - Use Cuda.
2020-07-15 17:08:40,220 - root - INFO - Namespace(balance_data=False, base_net=None, base_net_lr=0.0012, batch_size=5, checkpoint_folder='models_new/', dataset_type='voc', datasets=['/home/cjd/ssd.pytorch/data/VOCdevkit/VOC2007'], debug_steps=100, extra_layers_lr=None, freeze_base_net=False, freeze_net=False, gamma=0.1, lr=0.01, mb2_width_mult=1.0, milestones='80,100', momentum=0.9, net='mb3-ssd-lite', num_epochs=300, num_workers=0, pretrained_ssd='models_new/mb3-ssd-lite-Epoch-149-Loss-2.2059922739140037.pth', resume=None, scheduler='cosine', t_max=100.0, use_cuda=True, validation_dataset='/home/cjd/ssd.pytorch/data/VOCdevkit/VOC2007', validation_epochs=5, weight_decay=0.0005)
2020-07-15 17:08:40,233 - root - INFO - Prepare training datasets.
2020-07-15 17:08:40,240 - root - INFO - No labels file, using default VOC classes.
2020-07-15 17:08:40,246 - root - INFO - Stored labels into file models_new/voc-model-labels.txt.
2020-07-15 17:08:40,246 - root - INFO - Train dataset size: 3300
2020-07-15 17:08:40,246 - root - INFO - Prepare Validation datasets.
2020-07-15 17:08:40,248 - root - INFO - No labels file, using default VOC classes.
2020-07-15 17:08:40,248 - root - INFO - validation dataset size: 826
2020-07-15 17:08:40,248 - root - INFO - Build network.
2020-07-15 17:08:40,397 - root - INFO - Init from pretrained ssd models_new/mb3-ssd-lite-Epoch-149-Loss-2.2059922739140037.pth
2020-07-15 17:08:40,519 - root - INFO - Took 0.12 seconds to load the model.
2
2020-07-15 17:08:43,911 - root - INFO - Learning rate: 0.01, Base net learning rate: 0.0012, Extra Layers learning rate: 0.01.
2020-07-15 17:08:43,911 - root - INFO - Uses CosineAnnealingLR scheduler.
2020-07-15 17:08:43,911 - root - INFO - Start training from epoch 0.
